release-notes/v1.3.0.md

```diff
@@ -20,6 +20,9 @@ The following table lists the known issues for HPE CSI Driver for Kubernetes v1.
 |CON-1122|csi.k8s|iSCSI CHAP credentials applied are not reflected using iscsiadm command|When iSCSI CHAP settings are modified with `HPE CSI Driver`, credentials are updated using iscsiadm command when next pod is scheduled onto the node. However, these credentials are not displayed using iscsiadm command with already existing sessions|This is just cosmetic and current CHAP settings are updated with each iSCSI node entry under /etc/iscsi/nodes or /var/lib/iscsi/nodes directories. When iSCSI login is attempted again, latest values will be used by iscsid|
 |CON-1260|csi.k8s|iSCSI CHAP username/password is not updated in HPENodeInfos custom resource|HPE CSI driver maintains iSCSI CHAP credentials within each HPENodeInfo custom resource instance. When driver is re-deployed with changes to iSCSI CHAP username or password, sometimes HPENodeInfo instances are not updated/re-created causing them to contain stale data. This causes failures when an application pod is scheduled onto the node with stale iSCSI CHAP username or password|Possible workarounds for this include:<br />1. Delete HPENodeInfos instances manually and re-deploy CSI driver to create them with updated values.<br />2. Edit HPENodeInfos to update iSCSI CHAP username/password manually with each instance. Password has to be base64 encoded value|
 |CON-1266|csi.k8s|Failure to edit PVC size and other volume properties at the same time|With the introduction of the HPE Volume Mutator from v1.3.0, users can now edit volume parameters through PVC annotations. But, when volume properties are edited along with size at the same time, both the CSI Resizer and the HPE Volume Mutator might attempt PVC updates(i.e status and conditions). This might cause errors with either one of them during update saying resource version has changed externally and update process might fail|Avoid editing volume properties through annotations along with size at the same time. Resize is a different operation which involves controller/node expansion triggered through CSI external resizer. It is recommended to perform resize (volume size edit) and editing the rest of the volume properties like limitIops, limitMbps, etc.. as seperate operations.
+|CON-1099|csi.k8s|Editing PVC with alllowed mutations but missing overrides fails| Editing a PVC fails with incorrect error if a field is not allowed in "Overrides"  but allowed in "mutation" and present in PVC definition.|Mutation fields need to be present for overrides in storage class to prevent this error.
+|CON-1476|csi.k8s|PVCs parameters which are allowed in mutations and not in overrides are updated automatically|PVC with annotation for parameters which are allowed in "mutation" and not in "Overrides" gets updated automatically after about 2 minutes of time. Volumes get removed from volume collection as well.|Mutation fields need to be present for overrides in storage class to prevent this error.
+
```

Comment:
These don't seem to be showing up as 2 separate rows in the table.  Are you missing a | somewhere?

Reply:
Looks like all the "|" are present. Also in https://github.com/ckallianpur/csi-driver/blob/doc/release-notes/v1.3.0.md it shows it as separate rows.

Reply:
Click the "Display the rich diff" button at the top of this page.  Do you see 2 rows?

Reply:
updated